etc/clang-format-all.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+. "$(dirname "${BASH_SOURCE[0]}")/../.evergreen/init.sh"
```

Comment:
The `|| have_command cmd.exe` in:

https://github.com/mongodb/libmongocrypt/blob/73302cd485cd63becec92a29dff5f09f2731b4ab/.evergreen/init.sh#L145

was preventing my attempt to execute this script in WSL2:

```
Uname is [Linux <hostname> 5.15.79.1-microsoft-standard-WSL2 #1 SMP Wed Nov 23 01:01:46 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux]
```

It's not directly related to these changes, but can the check for `cmd.exe` be removed?

Reply:
Likely yes. I'll investigate that.

Reply:
The applied update does not seem to work as intended (case-sensitivity of `grep -q 'Microsoft'` failing to match `/proc/version` contents?), but given this is not directly related to this PR (and I can apply local workarounds anyways), I am not opposed to discarding these changes to simplify this PR.

Note: for reference, the following edit to `init.sh` seems to work:

```diff
- _uname="$(uname -a | tr '[:upper:]' '[:lower:]')"
+ _uname="$(uname | tr '[:upper:]' '[:lower:]')"

- if [[ "$_uname" =~ .*cywin|windows|mingw|msys.* ]] || have_command cmd.exe; then
+ if [[ "$_uname" =~ .*cywin|windows|mingw|msys.* ]] || (have_command cmd.exe && [[ ! "$_uname" =~ .*wsl.* ]]); then
```

Note 2: should `cywin` be `cygwin`?

Reply:
Thanks for checking and providing a proper fix. I'll remove these changes and put in a new PR later. And yes, that's a typo.

Reply:
Note: fixed diff to indicate _addition_ of `-a` to `uname` command rather than removal as otherwise `_uname` will simply be `linux`. The required "5.15.79.1-microsoft-standard-WSL2" portion containing "wsl" is part of the kernel release (`-r`/`--kernel-release`), not the kernel name (`-s`/`--kernel-name`, default. Using `-sr` instead of `-a` should therefore also suffice.